daemon/top_unix.go

```diff
@@ -130,20 +126,145 @@ func psPidsArg(pids []uint32) string {
 	return string(b)
 }
 
+// customFields checks whether o/-o/--format <field,...> option
+// was given to ps
+func customFields(args []string) bool {
+	/*
+	 * ps allows for a few fancy ways to provide an argument:
```

Comment:
nit: this comment could be part of the function's description instead of being in the function body

Reply:
I am confident this is the best place for it. The function description tells *what* the function does, while the comments inside the function body tells *how* exactly it does that. This particular comment, I believe, is crucial to understanding how the function code works, but a user of the function doesn't have to know such gory details :)

Of course, if a function user is a fellow developer, the boundary is not so strict... but anyway